### ISSUE TITLE
App crashes on macOS Mojave (needs to add NSPrincipalClass to Info.plist)

### ISSUE THREAD

**Issue by LumingYin** [rlxone/Equinox#9]
My Mac runs macOS Mojave (18G9323). After downloading Equinox from the Mac App Store, it crashes on launch with the following error message in Console:
```
2021-11-25 01:51:52.819 Equinox[10238:1025170] No Info.plist file in application bundle or no NSPrincipalClass in the Info.plist file, exiting
```

If I edit Info.plist and add the following key-value pair and re-sign the Equinox app, it launches as expected:
```
<key>NSPrincipalClass</key>
<string>NSApplication</string>
```

Can you please update the app to include the key-value pair above by default?

**Comment by rlxone** [comment]
Hi. Thank you for your feedback. 
I fixed this in [v1.0.1](https://github.com/rlxone/Equinox/releases/tag/v1.0.1) version.
Please update your app or compile [v.1.0.1](https://github.com/rlxone/Equinox/releases/tag/v1.0.1) version.

**Comment by LumingYin** [comment]
v1.0.1 works great, thank you for the fix!
